device-orientation.html

```diff
@@ -66,6 +75,10 @@
           beta: event.beta,
           gamma: event.gamma
         });
+
+        this._setSensorsAvailable(
+          !(event.alpha === null && event.beta === null && event.gamma === null)
```

Comment:
Shouldn't it be better to use JavaScript `Boolean()` function instead of `!` operator?

Should we care about `null === undefined // false`?